apps/datahub/src/app/record-preview-datahub/record-preview-datahub.component.css

```diff
@@ -0,0 +1,10 @@
+.record-item {
+  height: 10.5em;
+  cursor: pointer;
+}
+.publisher {
+  width: 15rem;
```

Comment:
Not tested but on the screenshot the width seems to vary from one record to the next

Reply:
yes it is because of the image size and shape... never really understood what is the best way to render thumnails so they always have the same size.. Using `width` and `height` attributes of the `img` tag ?

Reply:
There is a `gn-ui-record-thumbnail` component, see: https://github.com/geonetwork/geonetwork-ui/blob/master/libs/ui/search/src/lib/record-preview-card/record-preview-card.component.html

Maybe you'll have more luck with it :)